### PR TITLE
qa/tests: client-upgrade-luminous-octopus tests

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.ceph.com/teuthology-2020-03-25_01:31:08-upgrade:client-upgrade-luminous-octopus-octopus-distro-basic-smithi/

**Comment by yuriw** [comment]
removed "devstack" per chat with @dillaman as broken, e.g no `basic` tests anymore
http://pulpito.ceph.com/teuthology-2020-03-25_17:45:34-upgrade:client-upgrade-luminous-octopus-octopus-distro-basic-smithi/

File: qa/packages/packages.yaml [hunk 1]
```diff
@@ -8,7 +8,10 @@ ceph:
   - ceph-fuse
   - ceph-test
   - radosgw
-  - python-ceph
+  - python-rbd
```
Comment (tchaikov) [inline qa/packages/packages.yaml]: this does not look right. we don't package `python-*` anymore in octopus.

**Comment by yuriw** [comment]
running on @jdurgin commit => http://pulpito.ceph.com/teuthology-2020-03-27_00:58:05-upgrade:client-upgrade-luminous-octopus-octopus-distro-basic-smithi/

**Comment by yuriw** [comment]
http://pulpito.ceph.com/teuthology-2020-03-29_01:05:01-upgrade:client-upgrade-luminous-octopus-octopus-distro-basic-smithi/

**Comment by yuriw** [comment]
Fix should be available here: https://github.com/ceph/ceph/pull/34291
For:
in the client logs on smithi025 (http://pulpito.ceph.com/teuthology-
2020-03-29_01:05:01-upgrade:client-upgrade-luminous-octopus-octopus-distro-basic-smithi/)
Josh found:

2020-03-29T02:11:27.268+0000 7f3249ffb700 -1
librbd::image::OpenRequest: failed to refresh image: (95) Operation
not supported
(2020-03-29T02:11:27.268+0000 7f3249ffb700 -1 librbd::ImageState:
0x239e500 failed to open image: (95) Operation not supported

**Pull request merged**